data/functions.sql

```diff
@@ -1021,14 +1021,22 @@ BEGIN
       WHERE mu.wikidataid = wikidata_id;
   END IF;
 
-  -- finally, try states and provinces
+  -- try states and provinces
   IF NOT FOUND THEN
     SELECT
       min_label, max_label INTO min_zoom, max_zoom
       FROM ne_10m_admin_1_states_provinces sp
       WHERE sp.wikidataid = wikidata_id;
   END IF;
 
+  -- finally, try localities
+  IF NOT FOUND THEN
+    SELECT
+      rank_min, rank_max INTO min_zoom, max_zoom
```

Comment:
The rank_min here is a red herring... but we can use this slot to solve a different problem where the min_zoom between Natural Earth and OSM is inconsistent.

TO:
>  **min_zoom**, rank_max INTO min_zoom, max_zoom